docs/reference/team-processes/release-process.md

```diff
@@ -2,146 +2,177 @@
-
-All product icons live in a dedicated [assets directory](https://github.com/mozilla/fxa/tree/main/assets) in the fxa repo. This directory is independent of individual packages in the monorepo because it doesn't need to be wrapped up in any particular container, and these icons may be used across different servers.
-
-The `assets` directory is uploaded manually to the FxA CDN at https://accounts-static.cdn.mozilla.net as part of the release process. Subdirectories map directly to paths under the CDN domain: content in `assets/foo` maps to https://accounts-static.cdn.mozilla.net/foo.
+1. Ensure that any configuration changes noted in the deployment bug land in [cloudops-deployment][cloudops-deployment].
 
-The Stripe product icons live in `assets/product-icons`.
+1. Run any outstanding database migrations. These are applied automatically for `dev` and `stage` but are reviewed manually for production since we may need to take care with the changes to avoid slow queries.  The migrations should be included in the [deployment doc][deployment-doc].
 
-It's fine to create other subdirectories for new collections of assets as needed but you'll need to modify the push script to include them.
-
-The release script for static assets is `.github/workflows/upload-assets-to-cdn.yml`. 
+1. Build `fxa-auth`, `fxa-content`, `fxa-oauth`, `fxa-profile`, `fxa-verifier` in stage on Jenkins using git commit from cloudops-deployment PR and docker images referenced in deploy bug
 
-Icons and other assets are normally deployed with every release. If icons need to be deployed outside the regular release process, ask operations for help.
+1. If this is a push to production:
+    1. Ensure we have the appropriate QA signoffs. If major issues are found, a new patch is made and engineering will need to create a patch release
+    1. Deploy fxa-* to production
+    1. Close the initial deployment bug. It may be commented on later if there are patch releases in production.
 
-## FAQ
+1. Otherwise, if this is a staging tag, request QA on stage by posting in the bug.
 
-### What if the merge messes up the changelog?
+### Patch releases
 
-After merging but before pushing, you should check the changelog to make sure that the expected versions are listed and they're in the right order. If any are missing or the order is wrong, manually edit the changelog so that it makes sense, using the commit summaries from `git log --graph --oneline` to fill in any blanks as necessary.
+Patch (dot) releases are used between official releases to release code into staging or production that can't wait for the normal release cycle. What warrants a patch release may be evaluated on a case-by-case basis, but [S1 bugs][bug-severity] would likely drive one. If QA's signoff after testing a release on staging comes back yellow or red, we should check with engineering management for their recommended course of action, as we may need a dot release before pushing a release to production.
```

Comment:
Would love to also include these scenarios: Epic tasks needed to make a deliverable, Tasks needed to meet a RP launch, S1 & S2 severity bugs found during QA cycle (this is somewhat covered with the yellow report sentence).